monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/frontends.dashboard.py

```diff
@@ -0,0 +1,50 @@
+from grafanalib.core import (
+    Dashboard, Graph, Row, Target
+)
+from grr_grafanalib_dashboards.util import add_data_source
+from grr_grafanalib_dashboards.config import PANELS
+
+GRR_COMPONENT = "frontend"
+
+dashboard = Dashboard(
+    title="{}s Dashboard".format(GRR_COMPONENT).title(),
+    rows=[
+        Row(panels=[
+            panel(GRR_COMPONENT) for panel in PANELS
+        ]
+        ),
+        Row(panels=[
+          Graph(
+            title="QPS",
+            targets=[
+                Target(
+                    expr='sum(rate({}_request_count_total[1m]))'.format(GRR_COMPONENT),
+                    legendFormat="Requests",
+                ),
+            ],
+            ),
+            Graph(
+            title="Request Latency Rate",
+            targets=[
+                Target(
+                    expr='sum(rate({0}_request_latency_sum[5m])) / sum(rate({0}_request_latency_count[5m]))'.format(GRR_COMPONENT),
+                    legendFormat="Latency",
+                ),
+            ],
+            ),
+        ]),
+        Row(panels=[
+            Graph(
+                title="Active Tasks Count",
+                targets=[
+                    Target(
+                        expr='sum({}_active_count)'.format(GRR_COMPONENT),
```

Comment:
Same here: we know that we're inside the frontend code and the code is not meant to be reused for other components. Naming the metric literally would make this code easier to read, I think.

Basically, it makes sense to put a string literal into a constant if it's used in multiple places. This way it's easy to change it. But metrics names are fixed and so changing GRR_COMPONENT to a different value (i.e. "GRR Frontend") would just break them.